src/probnum/problems/zoo/filtsmooth/_ssm_examples.py

```diff
@@ -0,0 +1,328 @@
+import numpy as np
+
+from probnum import diffeq, filtsmooth, problems, randvars, statespace
+
+__all__ = ["car_tracking", "logistic_ode", "ornstein_uhlenbeck", "pendulum"]
+
+
+def car_tracking(delta_t=0.2, measvar=0.5, t_max=20.0):
+    r"""State space model for a simple car-tracking scenario.
+
+    A linear, time-invariant Gaussian state space model for car-tracking, based on
+    Example 3.6 in Särkkä, 2013. [1]_
+    Let :math:`X = (\dot{x}_1, \dot{x}_2, \ddot{x}_1, \ddot{x}_2)`. Then the state
+    space model has the following discretized formulation
+
+    .. math::
+
+        X(t_{n}) &=
+        \begin{pmatrix}
+          0 & 0 & 1 & 0 \\
+          0 & 0 & 0 & 1 \\
+          0 & 0 & 0 & 0 \\
+          0 & 0 & 0 & 0
+        \end{pmatrix} X(t_{n-1})
+        +
+        q(t_n) \\
+        y_{n} &=
+        \begin{pmatrix}
+          1 & 0 & 0 & 0 \\
+          0 & 1 & 0 & 0 \\
+        \end{pmatrix} X(t_{n})
+        + r_n
+
+    where :math:`q(t_n) \sim \mathcal{N}(0, Q)` and :math:`r_n \sim \mathcal{N}(0, R)`
+    for process noise covariance matrix :math:`Q` and measurement noise covariance
+    matrix :math:`R`.
+
+    Parameters
+    ----------
+    delta_t : float
+        The step size of the discretized time grid on which the model is considered.
+    measvar : float
+        The marginal measurement variance.
+    t_max : float
+        The time limit of the grid.
+
+    Returns
+    -------
+    dynmod
+        The `DiscreteLTIGaussian` dynamics model.
+    measmod
+        The `DiscreteLTIGaussian` measurement model.
+    initrv
+        A `Normal` random variable for the Gaussian initial conditions.
+    regression_problem
+        A `RegressionProblem` with time points and noisy observations.
+
+    References
+    ----------
+    .. [1] Särkkä, Simo. Bayesian Filtering and Smoothing. Cambridge University Press,
+        2013.
+
+    """
+
+    dynamat = np.eye(4) + delta_t * np.diag(np.ones(2), 2)
+    dynadiff = (
+        np.diag(np.array([delta_t ** 3 / 3, delta_t ** 3 / 3, delta_t, delta_t]))
+        + np.diag(np.array([delta_t ** 2 / 2, delta_t ** 2 / 2]), 2)
+        + np.diag(np.array([delta_t ** 2 / 2, delta_t ** 2 / 2]), -2)
+    )
+    measmat = np.eye(2, 4)
+    measdiff = measvar * np.eye(2)
+    mean = np.zeros(4)
+    cov = 0.5 * measvar * np.eye(4)
+
+    dynmod = statespace.DiscreteLTIGaussian(
+        state_trans_mat=dynamat, shift_vec=np.zeros(4), proc_noise_cov_mat=dynadiff
+    )
+    measmod = statespace.DiscreteLTIGaussian(
+        state_trans_mat=measmat,
+        shift_vec=np.zeros(2),
+        proc_noise_cov_mat=measdiff,
+    )
```

Comment:
I have a feeling you used this naming here because this is how it used to be done, but secretly prefer `dynamics_model` and `measurement_model`, right? :D If so, please go ahead and change it here. We use the longer names inside the filters now anyway, so there is no real reason for `dynmod` anymore.

Reply:
I would indeed prefer that, and I'll change it :)